Crash report: 
Exception: No module named 'PyQt6'
Traceback: 
  File "/Users/yesod/Desktop/Code/arizona/alora/maestro/app.py", line 864, in <module>
    run_with_crash_writing("MaestroApp", main)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/yesod/Desktop/Code/arizona/alora/maestro/scheduleLib/crash_reports.py", line 36, in run_with_crash_writing
    raise e
  File "/Users/yesod/Desktop/Code/arizona/alora/maestro/scheduleLib/crash_reports.py", line 33, in run_with_crash_writing
    function(*args, **kwargs)
    ~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/yesod/Desktop/Code/arizona/alora/maestro/app.py", line 27, in main
    from PyQt6 import QtGui, QtCore
